python/tvm/relay/frontend/tflite.py

```diff
@@ -616,6 +621,36 @@ def convert_greater(self, op):
                 'TFlite quantized greater operator is not supported yet.')
         return self._convert_elemwise(_op.greater, op)
 
+    def convert_greater_equal(self, op):
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                'TFlite quantized GREATER_EQUAL operator is not supported yet.')
```

Comment:
The previous operators seem to be all lowercase letters here. Maybe you should use `greater_equal` for consistency. The following operators are the same.

Reply:
From my perspective, when the operator is in lowercase it somehow mixes with the rest of the sentence so writing in capitals makes it a bit more clear...  imagine this is not the only error you get in the console. Anyway, I am open to suggestions :)

Reply:
Seems that we have two styles of it. Some code use `lowercase` like `convert_greater`, some code use `uppercase` like `convert_neg`. I think we should unify them to `uppercase`.  Two reasons: 1. it corresponds to our `convert_map` / tflite operator string represenation. 2. it could make us catch the operator more quickly in one sentence like @inadob said. So @inadob could you help to update other code to `uppercase` too? Thanks.